src/components/signin/sign-in-lazy.tsx

```diff
@@ -0,0 +1,8 @@
+import { FC, lazy, LazyExoticComponent } from 'react';
+
+import { routeWithSuspense } from '../../route-with-suspense';
+import Loader from '../loader/loader';
+
+const SigninLazy: LazyExoticComponent<FC> = lazy(() => import('./sign-in'));
```

Comment:
No need to specify such file type
```suggestion
const SigninLazy = lazy(() => import('./sign-in'));
```